lib/main.py

```diff
@@ -87,8 +87,8 @@ def set_up_check_in(arguments: List[str]) -> None:
         logger.error("Invalid arguments. For more information, try '--help'")
         sys.exit(2)
 
-    logger.debug(
-        "Monitoring %d accounts and %d reservations", len(config.accounts), len(config.reservations)
+    print(
```

Comment:
Did you mean to use print instead of logger.debug?

Reply:
I think it was meant to be print. It's helpful for debugging though so I'll make it a `logger.info`.